src/app_selection.rs

```diff
@@ -51,10 +51,13 @@ pub async fn try_get_valid_app(
         Some(app_info) => app_info,
         None => {
             let mut installed_apps = installed_apps.write().await;
-            let new_installed_apps = admin_call.list_apps().await.unwrap_or_else(|e| {
-                tracing::error!("Failed to get a list of apps from Holochain: {}", e);
-                vec![]
-            });
+            let new_installed_apps = admin_call
+                .list_apps(Some(AppStatusFilter::Running))
```

Comment:
This had been missed without a real admin websocket. We do actually want to pass a filter because there's no point connecting to an app that isn't running.